packages/ra-ui-materialui/src/input/AutocompleteArrayInput.tsx

```diff
@@ -1,627 +1,6 @@
-import React, {
-    useCallback,
-    useEffect,
-    useRef,
-    useMemo,
-    isValidElement,
-} from 'react';
-import { styled } from '@mui/material/styles';
-import Downshift, { DownshiftProps } from 'downshift';
-import classNames from 'classnames';
-import get from 'lodash/get';
-import { TextField, Chip, InputProps } from '@mui/material';
-import { TextFieldProps } from '@mui/material/TextField';
-import {
-    useInput,
-    FieldTitle,
-    ChoicesInputProps,
-    useSuggestions,
-    warning,
-    mergeRefs,
-} from 'ra-core';
-import debounce from 'lodash/debounce';
+import * as React from 'react';
+import { AutocompleteInput } from './AutocompleteInput';
 
-import { InputHelperText } from './InputHelperText';
-import { AutocompleteSuggestionList } from './AutocompleteSuggestionList';
-import { AutocompleteSuggestionItem } from './AutocompleteSuggestionItem';
-import { AutocompleteInputLoader } from './AutocompleteInputLoader';
-import {
-    SupportCreateSuggestionOptions,
-    useSupportCreateSuggestion,
-} from './useSupportCreateSuggestion';
-
-/**
```

Comment:
I think you should keep some jsDoc, at least to explain that it's deprecated

Reply:
Done